src/providers/patreon.js

```diff
@@ -0,0 +1,27 @@
+/** @type {import(".").OAuthProvider} */
+export default function Patreon(options) {
+    return {
+        id: 'patreon',
+        name: 'Patreon',
+        type: 'oauth',
+        version: '2.0',
+        authorization: { 
+          url : 'https://www.patreon.com/oauth2/authorize?response_type=code',
+          params: { scope: 'identity identity[email] identity.memberships', grant_type: 'authorization_code' }
```

Comment:
```suggestion
          params: { scope: 'identity identity[email] identity.memberships' }
```

I believe `grant_type` is set by default.

Also, what's the reason for the `identity.memberships` scope?

Do we need this to get either of these `id`, `name`, `image`, `email`?

If not, it should be best if we set as few scopes as possible by default.

Reply:
Indeed, I'll remove the unnecessary scope